helm/ingress-controller/values.yaml

```diff
@@ -203,3 +203,7 @@ log:
 ## ref: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/
 ##
 lifecycle: {}
+
+## @param bindings.enabled Whether to enable the Endpoint Bindings feature
+bindings:
+  enabled: true
```

Comment:
do you want this enabled by default during initial development phases?

Reply:
Nope, I'll go change that. Thank you.